apps/design-land/src/scss/component-themes.scss

```diff
@@ -1,4 +1,5 @@
-@use '../app/core/code-preview/component/code-preview-theme' as code-preview;
+// stylelint-disable-next-line max-line-length
+@use '@daffodil/docs-components/src/code-preview/component/code-preview-theme.scss' as code-preview;
```

Comment:
This doesn't work. I think `docs-components` package needs its own `scss` folder that has its own `component-theme.scss` and `styles.scss` files. @damienwebdev

Reply:
yeah I don't know what I'm doing for this. I changed some stuff that I think is closer to the solution but it doesn't work yet

Reply:
fixed it